README.rst

```diff
@@ -32,8 +32,8 @@ an asynchronous version of the Javascript code if possible.
 .. |coveralls| image:: https://coveralls.io/repos/jazzband/django-analytical/badge.svg
    :alt: Test coverage
    :target: https://coveralls.io/r/jazzband/django-analytical
-.. |health| image:: https://landscape.io/github/jazzband/django-analytical/master/landscape.svg?style=flat
-   :target: https://landscape.io/github/jazzband/django-analytical/master
+.. |health| image:: https://img.shields.io/codacy/grade/********************/master.svg
+   :target: https://www.codacy.com/app/jazzband/django-analytical
```

Comment:
Oooops! Can someone sign up django-analytical on Codacy? @jezdez maybe?

We can then get a proper badge from [the Codacy account](https://www.codacy.com/app/jazzband/django-analytical).

Reply:
This relates to #145, by the way.